### DISCUSSION TITLE
Gender (pronoun) detector using IETF data tracker

### DISCUSSION THREAD

**Discussion by sbenthall** [datactive/bigbang#525] [Ideas]
The IETF Data Tracker tool includes an (optional?) biography section for its record of people.

Example:

`Person(resource_uri=PersonURI(uri='/api/v1/person/person/116736/', params={}), id=116736, name='Niels ten Oever', name_from_draft='Niels ten Oever', ascii='Niels ten Oever', ascii_short='', user='', time=datetime.datetime(2014, 1, 16, 8, 11, 46), photo='https://www.ietf.org/lib/dt/media/photo/Niels_Ten_Oever-2-800px.jpg', photo_thumb='https://www.ietf.org/lib/dt/media/photo/Niels_Ten_Oever-2-800px_zkQNlnM.jpg', biography='Niels is a reviewer in the Human Rights Review Team, Secretary of the EDU team, the Editor of the Tao of the IETF, at-large member of the Internet IRSG, and former co-chair of the Human Rights Protocol Consideration Research Group.\r\n\r\nNiels is a PhD candidate in the Datactive Research Group at the Media Studies department at the University of Amsterdam. His research focuses on the evolution of the notion of public interest in the Internet architecture. He seeks to understand how invisible infrastructures provide a socio-technical ordering of our societies and how that might influence the distribution of wealth, power and possibilities.\r\n\r\nNiels has previously worked as Head of Digital at ARTICLE19, director of the Internet Protection Lab, and programme manager at Free Press Unlimited.', consent=True)`

This biography is a free text field with _pronouns_ in it:

> `'Niels is a reviewer in the Human Rights Review Team, Secretary of the EDU team, the Editor of the Tao of the IETF, at-large member of the Internet IRSG, and former co-chair of the Human Rights Protocol Consideration Research Group.\r\n\r\nNiels is a PhD candidate in the Datactive Research Group at the Media Studies department at the University of Amsterdam. His research focuses on the evolution of the notion of public interest in the Internet architecture. He seeks to understand how invisible infrastructures provide a socio-technical ordering of our societies and how that might influence the distribution of wealth, power and possibilities.\r\n\r\nNiels has previously worked as Head of Digital at ARTICLE19, director of the Internet Protection Lab, and programme manager at Free Press Unlimited.'`

Notice that "He" and "His" appear once each in this blurb, but not "She" or "Her".

As a way of augmenting the existing gender detecting tool, we can potentially us the biography blurb to extract gender pronouns for each IETF participant.

**Comment by npdoty** [comment]
I'm not sure how widespread these biography fields are likely to be. When I log in to the Datatracker, I don't even have the option of adding a bio.

It might be interesting to suggest that IETF start optionally collecting preferred pronouns in these profiles, but I don't see that yet.
